src/app/content/components/Toolbar.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { SFC } from 'react';
+import styled from 'styled-components';
+import {ListOl} from 'styled-icons/boxicons-regular/ListOl';
+import {Search} from 'styled-icons/fa-solid/Search';
+import {Print} from 'styled-icons/fa-solid/Print';
+import theme from '../../theme';
+
+const ListIcon = styled(ListOl)`
+  height: 2rem;
+  width: 2rem;
```

Comment:
according to the design this should be `width: 16px; height: 14px;` but i think maybe we should just set the height to `1.6rem` to match the font size and let the width figure itself out. it looks a little weird currently because you're still using the `boxicons` version instead of fa